torch/lib/c10d/ProcessGroupGloo.cpp

```diff
@@ -351,10 +351,11 @@ ProcessGroupGloo::SendWork::SendWork(
     std::unique_ptr<::gloo::transport::UnboundBuffer> buffer)
     : tensor_(tensor), buffer_(std::move(buffer)) {}
 
-void ProcessGroupGloo::SendWork::wait() {
+bool ProcessGroupGloo::SendWork::wait() {
+  bool sendCompleted;
```

Comment:
Initialize to false. Otherwise, its uninitialized value may be returned if `waitSend` throws.